### DISCUSSION TITLE
Confused how to use the extension

### DISCUSSION THREAD

**Discussion by reubster** [AltimateAI/vscode-dbt-power-user#203] [General] [answered]
I keep trying, I really want this to work and I do appreciate what @mdesmet is doing ... I just don't understand this extension. 

At the moment... nothing at all works. I've uninstalled and reinstalled. I've re-read the readme. I've cross-checked all the settings. I dunno. I'm baffled. 

Generate model from source gives me an error 

<img width="418" alt="image" src="https://user-images.githubusercontent.com/19515061/201623854-b2394b26-f091-41d7-a86a-ecdee81ed155.png">

Query results gives me an error 

<img width="1517" alt="image" src="https://user-images.githubusercontent.com/19515061/201624005-21b93fde-3842-4fb5-b015-f3b26eb90f80.png">

I don't know HOW to see the compiled query. Oh, just found that. Same error as above. 

The model directory is empty. 

<img width="1378" alt="image" src="https://user-images.githubusercontent.com/19515061/201621485-157d3695-60f0-489d-a570-f090d80c24ac.png">

**Comment by mdesmet** [comment]
The first suggestion is that you need to do `dbt deps`.

If you don't install the package. the project can't be compiled, so nothing will work.

**Comment by reubster** [comment]
Hiya, 

Yes, I know. Except it's in a dbt project that has 8 dependencies, all installed. 

<img width="237" alt="image" src="https://user-images.githubusercontent.com/19515061/201685533-1bf03680-663c-411f-baf6-0fcafad87473.png">

**Comment by reubster** [comment]
Ah, it works fine if I start vscode in the root folder of one of the dbt projects.

**Comment by mdesmet** [comment]
There must be something going wrong in the project discovery. Can you do a search for dbt_project.yml in the root directory and list all found files?

**Comment by mdesmet** [comment]
Logging and a warning for overlapping projects has been added in 0.9.1. Please provide log output and let us know if you receive the warning.

**Comment by mdesmet** [comment, marked as answer]
Fixed in 0.9.4
